### PR TITLE
changing stop_container to ssh flow

### PR DESCRIPTION
**Pull request by raskin-aleksandr**

### CODE REVIEW

**Comment by raskin-aleksandr** [comment]
@orius123  can you take a look

File: content/io/cloudslang/docker/containers/stop_container.sl [hunk 1]
```diff
@@ -30,39 +30,76 @@
...
+          - result: returnResult.replace("\n","")
+          - standard_out
+          - standard_err
+          - exception
+        navigate:
+          SUCCESS: SUCCESS
+          FAILURE: FAILURE
+          FAIL_VALIDATE_SSH: FAILURE
+
   results:
```
Comment (orius123) [inline content/io/cloudslang/docker/containers/stop_container.sl]: where are the outputs that the docs declare?

**Comment by raskin-aleksandr** [comment]
@orius123 can you take and look. I need this operarion to finish `clear_container`

File: content/io/cloudslang/docker/containers/stop_container.sl [hunk 2]
```diff
@@ -30,40 +30,79 @@
...
+          SUCCESS: SUCCESS
+          FAILURE: FAILURE
+          FAIL_VALIDATE_SSH: FAILURE
+
   outputs:
-    - container_ID: returnResult
-    - error_message: "'' if 'STDERR' not in locals() else STDERR if returnCode == '0' else returnResult"
+    - result
+
   results:
```
Comment (orius123) [inline content/io/cloudslang/docker/containers/stop_container.sl]: need to align docs

**Comment by orius123** [comment]
@raskin-aleksandr after aligning docs please merge

**Pull request merged**